### PR TITLE
refactor: change java unit tests to use only one thread

### PR DESCRIPTION
**Pull request by grianbrcom**
Add TestThreadPoolExecutor to be able to run test syncronously.

Signed-off-by: Anton Grigorev <anton.grigorev@broadcom.com>

### CODE REVIEW

**Review by temanbrcom — APPROVED** [review]
🐬

File: server/src/test/java/org/eclipse/lsp/cobol/service/utils/TestThreadPoolExecutorTest.java [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+import java.util.concurrent.Future;
+import java.util.concurrent.TimeUnit;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+/**
+ * Test {@link TestThreadPoolExecutor}
+ */
+class TestThreadPoolExecutorTest {
+  private static final int THE_MAGIC_NUMBER = 42;
```
Comment (asatklichov) [inline server/src/test/java/org/eclipse/lsp/cobol/service/utils/TestThreadPoolExecutorTest.java:29]: isn't it 41 ;)

**Review by asatklichov — APPROVED** [review]
🎲

File: server/src/test/java/org/eclipse/lsp/cobol/service/utils/TestThreadPoolExecutorTest.java (hunk 1 above)
Comment (grianbrcom) [reply server/src/test/java/org/eclipse/lsp/cobol/service/utils/TestThreadPoolExecutorTest.java:29]: I've taken it from google calculator =)
image

**Pull request merged**